src/test/java/com/microsoft/sqlserver/jdbc/datatypes/TVPWithSqlVariantTest.java

```diff
@@ -340,7 +340,7 @@ public void testNull() throws SQLServerException {
             tvp.addRow((Date) null);
         }
         catch (Exception e) {
-            assertTrue(e.getMessage().startsWith("Inserting null value with column"));
+            assertTrue(e.getMessage().startsWith("Use of TVPs containing null sql_variant columns is not supported."));
         }
```

Comment:
I would again recommend using same SQLServerResource > getResource() to compare error messages, so we don't have to change Strings at every place it is referenced. Would make sense to make a separate PR for this.